geopandas/geodataframe.py

```diff
@@ -459,6 +459,17 @@ def __getitem__(self, key):
             result.__class__ = DataFrame
         return result
 
+
+    def explode(self):
+        """
+        Overriden from GeoPandasBase, returns rest of the dataframe joined to 
+        exploded geometry column.
```

Comment:
Can you put here the a full docstring? (you can copy paste from the other implementation, and edit where needed) 
Because users are not much with the information it is overridden from GeoPandasBase (you could leave that as a normal comment though, as it is useful for developers)